src/xunit.v3.common/Reflection/ReflectionAttributeInfo.cs

```diff
@@ -90,6 +90,13 @@ public IReadOnlyCollection<_IAttributeInfo> GetCustomAttributes(string assemblyQ
 					list ??= new List<ReflectionAttributeInfo>();
 					list.Add(new ReflectionAttributeInfo(attr));
 				}
+				else if (attributeType.IsGenericTypeDefinition
```

Comment:
This new behavior probably needs to get documented into `_IAttributeInfo`.

Reply:
Done!